### ISSUE TITLE
Rule configuration is ignored if used "--path" parameter

### ISSUE THREAD

**Issue by yuriy-tolstoguzov** [realm/SwiftLint#1075]
I am configuring SwiftLint (version 0.15.0, installed from brew) on my project and see some inconsistency. When I use next command

```bash
swiftlint
```

It goes through all files and find zero warnings/errors. When I run swiftlint like that:

```bash
swiftlint lint --path path_to_file
```

It finds warning (namely: "warning: Force Unwrapping Violation: Force unwrapping should be avoided. (force_unwrapping)") even tho this rule is not enabled in `.swiftlint.yml`.

I tried next options:
* Use 
```
whitelist_rules:
  - closing_brace
```
* Use 
```
disabled_rules:
  - force_unwrapping
```
* Use next options to execute swiftlint:
  - `swiftlint lint --path path_to_file`
  - `swiftlint lint --config .swiftlint.yml --path path_to_file`
  - `swiftlint lint -- strict --config .swiftlint.yml --path path_to_file`

All this options ignore config and check this rule in specified file.

**Comment by yuriy-tolstoguzov** [comment]
I had another swiftlint config on `~/Desktop` and my project was at `~/Desktop/MyProject`. For some reason SwiftLint tries to merge those 2 configs and my config with only whitelisting rules does don't have any priority. It basically only adds more rules to check during merge.
Now the question is how to disable searching for more configs after first found or passed as parameter.

**Comment by Mozahler** [comment]
Similarly, I have two .yml files: easyPassLint.yml (no opt-ins, some rules disabled) and strictPassLint.yml (many opt-ins). I tried modifying the command in Xcode run scripts window, but always get the same results no matter which .yml I specify: (I know it finds the file - when it doesn't I get an error)
```
swiftlint --config /Users/Alex/Alex/Workspace/easyPassLint.yml     OR
swiftlint --config /Users/Alex/Alex/Workspace/strictPassLint.yml 
```

**Comment by Mozahler** [comment]
OK. For me, this is a duplicate of:
`--config` is ignored in directories that contains `.swiftlint.yml` #550
I renamed my .swiftlint.yml to swxftlint.yml and everything works as expected.
Thank you for a great product!

**Comment by yuriy-tolstoguzov** [comment]
Yep, looks like it
